src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,49 +1,130 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private T[] elementData;
+    private int arraySize;
+
+    public ArrayList() {
+        elementData = (T[]) new Object[DEFAULT_CAPACITY];
+    }
+
+    private T[] grow() {
```

Comment:
this method should return a new bigger array with all data
and this method can be void

Reply:
fix